### PR TITLE
Implement add role feature

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
This pull request is related to issue #11

### CODE REVIEW

File: myproject/supermarket_manager/views.py [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+
+    if request.method == "POST":
+        data = json.loads(request.body)
+        role_name = data.get("role_name", None)
+
+        if role_name is not None:
+            try:
+                role = Role(role_name=role_name)
+                role.save()
+                return JsonResponse({"message": "Role added successfully"})
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/views.py:28]: Missing status code.
And should define constants for messages.

**Comment by hothanhtuan1209** [comment]
image
image
image

**Comment by hothanhtuan1209** [comment]
image

File: myproject/supermarket_manager/views.py (hunk 1 above)
Comment (hothanhtuan1209) [reply myproject/supermarket_manager/views.py]: Done

File: myproject/supermarket_manager/constants.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Constant for message
+MESSAGE_ADDED = "Added successfully"
+MESSAGE_EXISTS = "Name already exists"
+MESSAGE_REQUIRED = "Name is required"
+MESSAGE_INVALID_METHOD = "Invalid request method"
```
Comment (cuongdoan-agilityio) [inline myproject/supermarket_manager/constants.py:5]: Should update naming.

**Review by cuongdoan-agilityio — APPROVED** [review]

**Pull request merged**